docs/docs/mlflow-3/index.mdx

```diff
@@ -0,0 +1,182 @@
+
+Specifically, you will be able to:
+- View Logged Models alongside key information, including code and weights as artifacts and configurable parameters that affect model behavior
+- Holistically track all performance metrics related to your logged models with links to training and evaluation jobs
+- Search for and rank existing models based on a set of certain criteria, such as performance metrics on training or evaluation datasets
+
+### Quickstart
+
+This quickstart example demonstrates how to use the LoggedModel lineage feature in MLflow 3.0 with openai sdk, showcasing its integration with runs and traces.
+
+Prerequisite: ``pip install mlflow==3.0.0.rc0 openai``
+
+```python
+from openai import OpenAI
+
+import mlflow
+from mlflow.metrics.genai import answer_correctness, answer_similarity, faithfulness
+
+# turn on autologging for automatic tracing
+mlflow.openai.autolog()
+
+client = OpenAI()
+
+mlflow_ground_truth = (
+    "MLflow is an open-source platform for managing "
+    "the end-to-end machine learning (ML) lifecycle. It was developed by Databricks, "
+    "a company that specializes in big data and machine learning solutions. MLflow is "
+    "designed to address the challenges that data scientists and machine learning "
+    "engineers face when developing, training, and deploying machine learning models."
+)
+# Define evaluation metrics
+metrics = {
+    "answer_similarity": answer_similarity(model="openai:/gpt-4o"),
+    "answer_correctness": answer_correctness(model="openai:/gpt-4o"),
+    "faithfulness": faithfulness(model="openai:/gpt-4o"),
+}
+question = "What is MLflow?"
+
+with mlflow.start_run():
+    response = (
+        client.chat.completions.create(
+            messages=[{"role": "user", "content": question}],
+            model="gpt-4o-mini",
+            temperature=0.1,
+            max_tokens=2000,
+        )
+        .choices[0]
+        .message.content
+    )
+
+    # Calculate metrics based on the input, response and ground truth
+    # The evaluation metrics are callables that can be invoked directly
+    answer_similarity_score = metrics["answer_similarity"](
+        predictions=response, inputs=question, targets=mlflow_ground_truth
+    ).scores[0]
+    answer_correctness_score = metrics["answer_correctness"](
+        predictions=response, inputs=question, targets=mlflow_ground_truth
+    ).scores[0]
+    faithfulness_score = metrics["faithfulness"](
+        predictions=response, inputs=question, context=mlflow_ground_truth
+    ).scores[0]
+
+    # Fetch the LoggedModel that's automatically created during autologging
+    logged_model = mlflow.last_logged_model()
+    # Log metrics and pass model_id to link the metrics
+    mlflow.log_metrics(
+        {
+            "answer_similarity": answer_similarity_score,
+            "answer_correctness": answer_correctness_score,
+            "faithfulness": faithfulness_score,
+        },
+        model_id=logged_model.model_id,
+    )
```

Comment:
Is linking by `model_id` necessary? I thought MLflow would automatically figure this out based on the logged model

Reply:
It's necessary right now because the model is not logged inside a evaluation run, it was created by autologging during invocation